msteams-platform/sbs-file-handling-in-bot.yml

```diff
@@ -0,0 +1,314 @@
+### YamlMime:Tutorial
+title: Teams File Upload Bot
+metadata:
+  title: Teams File Upload Bot 
+  description: In this tutorial, you'll learn how to upload files to Teams from a bot and how to receive a file sent to a bot as an attachment.
+  audience: Developer
+  level: Beginner
+  ms.date: 28/12/2021
+  ms.topic: interactive-tutorial
+  nextTutorialHref: bots/how-to/bots-filesv4.md
+  nextTutorialTitle: Read more on send and receive files through the bot
+  ms.custom: mvc
+  ms.localizationpriority: none
+items:
+- durationInMinutes: 1
+  content: |
+   The bot is created using Bot Framework. It shows how to upload files to Teams from a bot and how to receive a file sent to a bot as an attachment. 
+   It also shows how to fetch inline images sent in message.
+    
+   This step-by-step guide helps you to do Teams file upload bot at Microsoft Teams. You'll see the following output:
```

Comment:
This step-by-step guide helps you to upload files from a bot to Microsoft Teams.....

Reply:
changes made

Reply:
changes made